src/sql/src/kafka_util.rs

```diff
@@ -117,9 +135,9 @@ fn extract(
 /// - If any of the values in `with_options` are not
 ///   `sql_parser::ast::Value::String`.
 pub fn extract_config(
```

Comment:
this seems like it was already being used for `CREATE SINK` as well as `CREATE SOURCE` - might be nice to amend the comment to reflect that

Reply:
Thanks! I missed that 🙂